index.html

```diff
@@ -107,155 +107,6 @@ <h4>I’am a Software Developer</h4>
-              <p>
-                A daily selection of privately personalized reads; no accounts
-                or sign-ups required. has been the industry's standard dummy
-                text ever since the 1500s, when an unknown printer took a
-                standard dummy text.
-              </p>
-              <ul>
-                <li class="tags">css</li>
-                <li class="tags">html</li>
-                <li class="tags">Bootstrap</li>
-                <li class="tags">Ruby</li>
-              </ul>
-              <button type="button" class="btn-desk btn btn-feature">
-                See Project
-              </button>
-            </div>
-          </div>
-
-          <ul id="Card-container">
-            <li class="project-card">
-              <div class="Card-description">
-                <h3>Profesional Art Printing Data</h3>
-                <p>
-                  A daily selection of privately personalized reads; no accounts
-                  or sign-ups required. has been the industry's standard
-                </p>
-                <ul class="Description-tags">
-                  <li>html</li>
-                  <li>Bootstrap</li>
-                  <li>Ruby</li>
-                </ul>
-              </div>
-              <button
-                id="btn-first"
-                class="btn-desk btn btn-card"
-                type="button"
-              >
-                See Project
-              </button>
-            </li>
-            <li class="project-card">
-              <div class="Card-description">
-                <h3>Profesional Art Printing Data</h3>
-                <p>
-                  A daily selection of privately personalized reads; no accounts
-                  or sign-ups required. has been the industry's standard
-                </p>
-                <ul class="Description-tags">
-                  <li>html</li>
-                  <li>Bootstrap</li>
-                  <li>Ruby</li>
-                </ul>
-              </div>
-              <button
-                id="btn-first1"
-                class="btn-desk btn btn-card"
-                type="button"
-              >
-                See Project
-              </button>
-            </li>
-            <li class="project-card">
-              <div class="Card-description">
-                <h3>Profesional Art Printing Data</h3>
-                <p>
-                  A daily selection of privately personalized reads; no accounts
-                  or sign-ups required. has been the industry's standard
-                </p>
-                <ul class="Description-tags">
-                  <li>html</li>
-                  <li>Bootstrap</li>
-                  <li>Ruby</li>
-                </ul>
-              </div>
-              <button class="btn-desk btn btn-card" type="button">
-                See Project
-              </button>
-            </li>
-            <li class="project-card">
```

Comment:
- [ ] great job making the changes requested by the previous reviewer. please make your popups a little bit smaller so the scrolling can disappear and it can be visible.
![Screenshot (150)](https://user-images.githubusercontent.com/65298048/174115853-730ee51c-745a-4dee-ad01-bda7ab1acbd8.png)